### PR TITLE
Add more icon stuff

### PR DESCRIPTION
**Pull request by bskiff**

### CODE REVIEW

**Comment by bskiff** [comment]
Do not merge - need to figure out Git LFS

File: index.html [hunk 1]
```diff
@@ -4,6 +4,7 @@
     <meta charset="utf-8" />
     <script defer type="module" src="/src/index.tsx"></script>
     <link rel="manifest" href="/manifest.json" />
+    <link rel="icon" type="image/png" href="/icons/icon-72x72.png" />
```
Comment (dskiff) [inline index.html:7]: we also need the the crazy, non-compliant thing for ios
https://developer.apple.com/library/archive/documentation/AppleApplications/Reference/SafariWebContent/ConfiguringWebApplications/ConfiguringWebApplications.html

File: src/components/PuzzleCell.tsx [hunk 2]
```diff
@@ -21,6 +21,7 @@ export const PuzzleCell = (props: Props): JSX.Element => {
   return (
     <div className="grid-cell-wrapper">
       <div
+        contentEditable="true"
```
Comment (dskiff) [inline src/components/PuzzleCell.tsx:24]: ?

**Review by dskiff — APPROVED** [review]

File: index.html (hunk 1 above)
Comment (bskiff) [reply index.html:7]: Sigh. Thanks Apple

File: src/components/PuzzleCell.tsx (hunk 2 above)
Comment (bskiff) [reply src/components/PuzzleCell.tsx:24]: This causes the on-screen keyboard to appear on mobile. It's still a little wonky though. Multiple characters can be entered in a single cell from mobile. Not sure why. I'm guessing it's something about the key up event.

Comment (bskiff) [reply src/components/PuzzleCell.tsx:24]: Or maybe the issue is caused by the content editable. Does that allow editing the content within the div directly, or something?

Comment (bskiff) [reply src/components/PuzzleCell.tsx:24]: I should read what it actually does

**Pull request merged**